schemas/README.md

```diff
@@ -453,6 +593,160 @@ Image format
 </tr>
 </table>
 
+## CubePrimitive
+
+A primitive representing a cube or rectangular prism
+
+<table>
+  <tr>
+    <th>field</th>
+    <th>type</th>
+    <th>description</th>
+  </tr>
+<tr>
+<td><code>pose</code></td>
+<td>
+
+[Pose](#pose)
+
+</td>
+<td>
+
+Position of the center of the cube and orientation of the cube
+
+</td>
+</tr>
+<tr>
+<td><code>size</code></td>
+<td>
+
+[Vector3](#vector3)
+
+</td>
+<td>
+
+Size of the cube along each axis
+
+</td>
+</tr>
+<tr>
+<td><code>color</code></td>
+<td>
+
+[Color](#color)
+
+</td>
+<td>
+
+Color of the arrow
+
+</td>
+</tr>
+<tr>
+<td><code>metadata</code></td>
```

Comment:
I don't really see the point of all this metadata in the primitives, nor do I understand how we would even show it to the user since I assume clicking a SceneEntity would select that entire SceneEntity

Reply:
>  I assume clicking a SceneEntity would select that entire SceneEntity

This would be a regression from the old3d panel behavior where you can select a single cube in a cube list, or a single point in a point cloud.

Note: in verifying this claim I realized there's a bug(?)missing feature(?) in old3d where the specific clicked cube in a cube list is not displayed. However, the index is available in the code and I guess is just being ignored. I think we may have changed this at some point.

<img width="590" alt="image" src="https://user-images.githubusercontent.com/14237/183469035-0328b64a-20a9-41c2-b4dc-f22b56e05fa1.png">

Reply:
Interesting. If we're going to make clicks select the individual primitives, then what is the point of having a SceneEntity? I thought the whole idea was so that I could build an "entity" out of primitives, and have it behave like one "object" in the scene.

I guess another option is it could behave like a "grouped object" in google slides, or we could let you hold a modifier key if you want to select individual primitives rather than the whole entity.